components/atom/textarea/src/index.js

```diff
@@ -0,0 +1,126 @@
+import React, {Fragment, Component} from 'react'
+import PropTypes from 'prop-types' // eslint-disable-line no-unused-vars
+import cx from 'classnames'
+
+import AtomHelpText from '@s-ui/react-atom-help-text'
+import AtomValidationText, {
+  AtomValidationTextTypes
+} from '@s-ui/react-atom-validation-text'
+import AtomLabel, {AtomLabelTypes} from '@s-ui/react-atom-label'
+
+const BASE_CLASS = 'sui-AtomTextarea'
+const SIZES = {
```

Comment:
kind of weird to me to use "short" "long" when we're using "big" "small" in other components.
But I know it's the vocabulary written in the spec ^^